R/runCCA.R

```diff
@@ -186,10 +229,10 @@ setMethod("calculateCCA", "SummarizedExperiment",
 #' @rdname runCCA
 #' @importFrom SingleCellExperiment reducedDim<-
 setMethod("runCCA", "SingleCellExperiment",
-    function(x, ..., altexp = NULL, name = "CCA")
+    function(x, ..., altExp = NULL, name = "CCA")
     {
-        if (!is.null(altexp)) {
-          y <- altExp(x, altexp)
+        if (!is.null(altExp)) {
+          y <- altExp(x, altExp)
```

Comment:
Replaced altexp with altExp since we use similar argument with uppercasing in other function (Is it ok to replace like this without support for altexp?)

Reply:
I think it is ok now. Not wide user base yet.